scripts/dopple.py

```diff
@@ -0,0 +1 @@
+dopple/dopple.py
```

Comment:
does it skip command line arguments?

Reply:
This is a symlink, github shows it strangely.